sdk/textanalytics/tests.yml

```diff
@@ -9,5 +9,6 @@ jobs:
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        AZURE_TEXT_ANALYTICS_ENDPOINT: $(java-textanalytics-sdk-endpoint)
+        AZURE_TEXT_ANALYTICS_ENDPOINT: $(java-azure-textanalytics-sdk-endpoint)
```

Comment:
You'll want to change this to the below, right now this is still looking at the value being loaded from the KeyVault.

```yml
AZURE_TEXT_ANALYTICS_ENDPOINT: $(AZURE_TEXT_ANALYTICS_ENDPOINT)
```

I may be mis-remembering this but @danieljurek mentioned that this may not need to be listed since the generated output uses the same name as the environment variable expected and it can automatically map the value.

Reply:
Yes, once the ARM template changes are correct. This yml file should no longer have these variables mentioned. Keeping these changes currently to help the service team investigate a bug on the Live tests pipeline that is using this PR.